packages/@aws-cdk/cloudformation-include/lib/private/yaml-cfn.ts

```diff
@@ -0,0 +1,60 @@
+import * as yaml from 'yaml';
+import * as yaml_cst from 'yaml/parse-cst';
+import * as yaml_types from 'yaml/types';
+
+/**
+ * Serializes the given data structure into valid YAML.
+ *
+ * @param obj the data structure to serialize
+ * @returns a string containing the YAML representation of {@param obj}
+ */
+export function serialize(obj: any): string {
```

Comment:
How about moving this to 'core' and reduce the dupes?

Reply:
CLI still cannot depend on `core` so doesn't help that much.

Also it should not be part of the public API, this package is/was an implementation detail.

Reply:
(not a strong opinion but) put it in `cx-api` but not export it? Usage would be -

```ts
import * as yaml_cfn from '@aws-cdk/cx-api/yaml-cfn'
```

Reply:
We have tricks/hacks for making things in `core` not public. See [`cfn-parse.ts`, for example](https://github.com/aws/aws-cdk/blob/c5a2addcd87ebb810dcac54c659fa60786f9d345/packages/%40aws-cdk/core/lib/cfn-parse.ts#L26-L26).